package/kernel/mac80211/files/lib/wifi/mac80211.sh

```diff
@@ -77,15 +87,22 @@ detect_mac80211() {
 		[ "$found" -gt 0 ] && continue
 
 		mode_band="g"
-		channel="11"
+		channel="6"
 		htmode=""
 		ht_capab=""
+		ssid="$secure_ssid"
+		encryption="psk2"
+		key="$secure_psk"
+		tx_power="5"
+		disabled="0"
```

Comment:
Changing the channels should go into a separate commit.
You are changing defaults here, not only values for the special case you want to treat.
So, 99.x % normal user will have to suffer from reduced tx_power in initial settings compared to 0.y % users benefitting from your changes.
Please find a way to make your introduced changes optional, so they only apply for the devices where they are needed.
Despite that, note that setting tx_power device-independently is a bad idea, as it ignores antenna_gain.
There are e.g. devices with 18 dB antennas, and countries with 20 dB max. For those, you would be over the limit (although I suspect the tx_power won't be actually applied then).

Reply:
I think this point is covered now.